easybuild/easyconfigs/p/PyTorch/PyTorch-1.2.0-fosscuda-2019a-Python-3.7.2.eb

```diff
@@ -0,0 +1,315 @@
+    '98af442368142a05acc2130289b2a31036e43dbba6b5ec7626e9fb2464ddceae',  # QNNPACK-20181227.tar.gz
+    '0054292529c523bd1fa0be942cd49a6f87061c66e9b0235bda0b1d5f53b80539',  # fbgemm-20190723.tar.gz
+    '9c59802895a7812d6ce63f25abf9ad75fe651ab7b550d79cd266fee34bfedfeb',  # asmjit-20171211.tar.gz
+    '3f2dc1970f397a0e59db72f9fca6ff144b216895c1d606f6c94a507c1e53a025',  # cpuinfo-20190201.tar.gz
+    'e99b904983d08ac8e9bddb5b0d21196b78ad9499e3c5d12192cee2ddd2b7515c',  # googletest-20180920.tar.gz
+    '0de43b6eaddd356f1d6cd164f73f37faf2f6c96fd684e1f7ea543ce49c1d144e',  # benchmark-20180606.tar.gz
+    '2ec954f18cec50a7063a7358ce555f7e11788a7f6d4e7e597d83687dc2f3b989',  # eigen-20190126.tar.gz
+    'fdd6f08bdb33d33f4df516ffb91730fdb163479c19502cfc983083fd9cf43bfa',  # gemmlowp-20181127.tar.gz
+    'ef8b624ef28e9e2d0e162065932dd2f25de8d6d1dc58df91bd36977e4bc0224c',  # nccl-20190625.tar.gz
+    '7224b03af4acbc54525105bb42e3ecd75c66a3a5b47e8a725ab008467c4109f9',  # cub-20170829.tar.gz
+    'b444b4428adc4befc777d9a1af2fd9bbf5f5a2339913668357196c5133e9cc7b',  # foxi-20190424.tar.gz
+    'ca3da1088102225e8f622853553f21cc8eaa328748ad39d382fa3a608f260470',  # ideep-20190701.tar.gz
+    'dc0a8d8d96cb8765782aa6ac1b509ad4db955d9bbb58fa5cc2265f0292756d72',  # tbb-20181009.tar.gz
+    '815b0397cd4be02a302eca4072362aee92fa3f3ad594f7d43f8838c0bd3043d8',  # mkl-dnn-20190316.tar.gz
+    'f4129843d5c2996419f96f10928edd02b2150998861a088dc7cfa1b6a058102a',  # mklml_lnx_2019.0.3.20190220.tgz
+    '001c9bf604aebe4b39ccad15332a71130b07b780c539ceca84d6c64cd6fc8a68',  # PyTorch-1.2.0_fix-findAVX.patch
+    'c4183bcb29a8bcbadea0341e93a3a32afdf860aa31331b768e787d899183da92',  # PyTorch-1.2.0_disable-tests-ppc64le.patch
+    # PyTorch-1.2.0_disable-tests-sandybridge.patch
+    '279a267fc5cee73e93d0ab1f8b88f61d199bc4ad8692b0ba28ede42c9f066fd4',
+    'b6eeaf87c4bdadf6c2543a6b30fb4fe384ef96e10b7fce67a6159dfb8ec94003',  # PyTorch-1.2.0_fix_ideep_avx2_issue.patch
+    'c37dd508d41e0bbecb4577b319dab1abc7c6eb9a97e134f65d8bbad88ddfa40e',  # PyTorch-1.2.0_fix_slogdet_issue.patch
+]
+
+builddependencies = [
+    ('CMake', '3.13.3'),
+    ('hypothesis', '4.23.4'),
+    ('psutil', '5.6.3'),
+    ('Ninja', '1.9.0'),
+]
+
+dependencies = [
+    ('Python', '3.7.2'),
+    ('SciPy-bundle', '2019.03'),
+    ('PyYAML', '5.1'),
+    ('MPFR', '4.0.2'),
+    ('GMP', '6.1.2'),
+    ('numactl', '2.0.12'),
+    ('FFmpeg', '4.1.3'),
+    ('cuDNN', '7.4.2.24'),
+    ('magma', '2.5.1'),
+    ('gflags', '2.2.2'),
+    ('glog', '0.4.0'),
+    ('Pillow', '6.0.0'),
+]
+
+download_dep_fail = True
+
+prebuildopts = 'PYTORCH_BUILD_VERSION=%(version)s PYTORCH_BUILD_NUMBER=1 VERBOSE=1 '
+prebuildopts += 'LDFLAGS="$LDFLAGS -ldl" '
+prebuildopts += 'USE_FFMPEG=ON USE_GLOO_IBVERBS=1 USE_GFLAGS=ON USE_GLOG=ON '
+prebuildopts += 'CUDNN_LIB_DIR=$EBROOTCUDNN/lib64 CUDNN_INCLUDE_DIR=$EBROOTCUDNN/include '
+prebuildopts += 'TORCH_CUDA_ARCH_LIST="3.5 6.0 7.0" '
```

Comment:
@branfosj why limit to only these 3 cuda archs here?
In PyTorch-1.0.1-fosscuda-2018b-Python-3.6.6.eb we have: `TORCH_CUDA_ARCH_LIST="3.0 3.2 3.5 3.7 5.0 6.0 6.1 7.0 7.2"`
In the Dockerfile of v1.2.0: `TORCH_CUDA_ARCH_LIST="3.5 5.2 6.0 6.1 7.0+PTX"`